### PR TITLE
Remove issues-count panel from the Issues-Tree

### PR DESCRIPTION
**Pull request by barbelity**
- [x] All [tests](https://ci.appveyor.com/project/jfrog-ecosystem/jfrog-idea-plugin) passed. If this feature is not already covered by the tests, I added new tests.
-----

### CODE REVIEW

File: src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java [hunk 1]
```diff
@@ -131,7 +121,7 @@ private JComponent createComponentsDetailsView(boolean supported) {
         JLabel title = new JBLabel(" Component Details");
         title.setFont(title.getFont().deriveFont(TITLE_FONT_SIZE));
 
-        issuesDetailsPanel = new JBPanel(new BorderLayout()).withBackground(UIUtil.getTableBackground());
+        issuesDetailsPanel = new JBPanel<>(new BorderLayout()).withBackground(UIUtil.getTableBackground());
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java:124]: To get the whole line selected:
```suggestion
        issuesDetailsPanel = new JBPanel<>(new GridLayout()).withBackground(UIUtil.getTableBackground());
```

File: src/main/java/com/jfrog/ide/idea/actions/CollapseAllAction.java [hunk 2]
```diff
@@ -31,19 +28,8 @@ public CollapseAllAction(@NotNull JTree tree) {
 
     @Override
     public void actionPerformed(AnActionEvent e) {
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/actions/CollapseAllAction.java:30]:
```suggestion
    public void actionPerformed(@NotNull AnActionEvent e) {
```

File: src/main/java/com/jfrog/ide/idea/actions/ExpandAllAction.java [hunk 3]
```diff
@@ -30,19 +28,8 @@ public ExpandAllAction(@NotNull JTree tree) {
 
     @Override
     public void actionPerformed(AnActionEvent e) {
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/actions/ExpandAllAction.java:30]:
```suggestion
    public void actionPerformed(@NotNull AnActionEvent e) {
```

File: src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java (hunk 1 above)
Comment (barbelity) [reply src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java:124]: Applied your suggestion in line 88.

**Pull request merged**